.travis.yml

```diff
@@ -1,18 +1,20 @@
 language: cpp
+os: osx
+osx_image:
+  - xcode10.1
+  - xcode11.2
+env:
+  global:
+  - MAKEFLAGS="-j2"
```

Comment:
I'd prefer not to have this either, if you want to compile it faster install ninja and use that i guess

Reply:
Ok, I just checked the Travis CI docs and saw that every instance has 2 cores.